samples/monster_generated.h

```diff
@@ -25,6 +25,14 @@ enum Color {
   Color_MAX = Color_Blue
 };
 
+inline Color (&EnumValuesColor())[3] {
```

Comment:
I was referring to the unusual function declaration syntax, as opposed to say `inline const Color *EnumValuesColor() {` or similar. I've frankly never seen anyone use `()` around the function signature, or `[]` after it. Can this be made more readable? I agree that we don't want to rely on std::array. Again, not sure why you can't just use `_MIN` and `_MAX`.